Pipfile

```diff
@@ -0,0 +1,49 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+# Most of the dependencies in here are for serveradmin and not actually
+# required for adminapi.
+
+# Adminapi:
+# Like ipaddress but for MACs
+netaddr = "*"
+
+# Serveradmin:
+# WSGI server
+gunicorn = "*"
+# Last version supporting python3.4
+Django = "~=1.11.0"
+# postgres lib used by django
+"psycopg2-binary" = "*"
+# Django support for network address fields on PostgreSQL
+django-netfields = "*"
+# Required to generate sprites from Graphite
+Pillow = "*"
+# Used to list loaded api addons at /api/functions
+# TODO: Drop this dependency
+django-markup-deprecated = "*"
+# TODO: What is this for?
+docutils = "*"
```

Comment:
Those 2 are probably not needed anymore.  The docs used to be rendered by Serveradmin, but I switched them to a link to GitHub.

Reply:
django-markup-deprecated is still needed at least. There is this page showing which api functions were loaded from external apps. That makes use of it. I'm not sure about docutils but I could check if that page still works without docutils. Thanks for the hint.